src/txmempool.h

```diff
@@ -704,7 +704,10 @@ class CTxMemPool
     /** Adds a transaction to the unbroadcast set */
     void AddUnbroadcastTx(const uint256& txid) {
         LOCK(cs);
-        m_unbroadcast_txids.insert(txid);
+        /** Sanity Check: the transaction should also be in the mempool */
```

Comment:
hm, is there a reason this is a [doxygen comment](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-doxygen-compatible-comments)? I'm not sure what would happen, but I suspect you want normal `//` notation here

Reply:
> I suspect you want normal `//` notation here

Agree, perhaps can be touched up in #18807